test/ruby/test_rubyoptions.rb

```diff
@@ -229,16 +229,16 @@ def test_version
 
     if JITSupport.supported?
       [
-        %w(--version --jit),
-        %w(--version --enable=jit),
-        %w(--version --enable-jit),
+        %w(--version --disable-yjit --jit),
+        %w(--version --disable-yjit --enable=jit),
+        %w(--version --disable-yjit --enable-jit),
```

Comment:
There's a couple of places where you added `--disable-yjit` in this file. Just double-checking, is that necessary?

Reply:
Ya, I think it's necessary.  I think those tests are specifically testing that we can enable MJIT.